workflow/defaults/default_resources.yaml

```diff
@@ -11,10 +11,10 @@
 
 godas_resource_table:
 #                ranks  ppn  wallclock              threads MB
-  r3dvar:       [  20,   20, !timedelta "01:00:00", 12,  "3072" ]
-  r3denvar:     [  80,   20, !timedelta "01:00:00", 12,  "3072" ]
-  obs_prep:     [  12,   12, !timedelta "01:00:00", 1,   "3072" ]
-  da_prep:      [   1,    1, !timedelta "00:30:00", 1,   "3072" ]
+  r3dvar:       [  24,    24, !timedelta "00:10:00", 12,  "3072" ]
+  obs_prep:     [  12,    12, !timedelta "00:10:00", 1,   "3072" ]
+  da_prep:      [  1,      1, !timedelta "00:30:00", 1,   "3072" ]
+  letkf_prep:   [  24,    24, !timedelta "00:10:00", 12,  "3072" ]
```

Comment:
Timing change: This means we will have to change the settings when we want to do science. I understand why, but something to be aware of.
Also, I may be wrong, but I think asking for 24 pe will translate to 2 nodes on hera, as opposed to 1 node when asking for 20 workers ... I'll check.

Reply:
- Yes, the timing change is to accommodate the conversion of the raw observations to ioda format.

- On hera, there are 40ppn, so there is no issue requesting 24p on one node.

Reply:
The LETKF prep is the socahofx3d, so we can use the resources for the r3dvar. I updated this at the reviewed version of the code.